src/mux-analytics.brs

```diff
@@ -894,7 +1066,11 @@ function muxAnalytics() as Object
 
       if m.video <> Invalid
         if m.video.content <> Invalid
-          m._videoContentProperties = m._getVideoContentProperties(m.video.content)
+          if m.video.contentIsPlaylist
+            m._videoContentProperties = m._getVideoContentProperties(m.video.content.getChild(m.video.contentIndex))
```

Comment:
we have a concern from one customer that the number of rendezvous we have is too large; accordingly, do we need to call this dynamically each time, or is there a way to not do that?